prac_05/wimbleton.py

```diff
@@ -0,0 +1,49 @@
+"""
+Wimbledon
+Estimate: 20 minutes
+Actual:   25 minutes
+
+"""
+FILENAME = "wimbledon.csv"
+INDEX_COUNTRY = 1
+INDEX_CHAMPION = 2
+
+
+def main():
+    """Read csv file, process data and print details about Wimbledon champions and countries."""
+    records = get_records(FILENAME)
+    champion_to_count, countries = process_records(records)
+    display_results(champion_to_count, countries)
+
+
```

Comment:
Sorry for the late reply :(, but good work for all the exercises, i think the commit messages are useful as well.
If possible you could also make comments in your code to show your thought process (but of course it's not needed for the practical itself )